ThunderCatsTests/UIAlertControllerTests.m

```diff
@@ -0,0 +1,66 @@
+//
+//  UIAlertControllerTests.m
+//  ThunderCats
+//
+//  Created by Abbey Jackson on 2016-05-09.
+//  Copyright © 2016 Metova. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "UIAlertController+TCAdditions.h"
+
+@interface UIAlertControllerTests : XCTestCase
+
+@end
+
+@implementation UIAlertControllerTests
+
+- (void)testNotImplementedAlert
+{
+    UIAlertController *alert = [UIAlertController tc_notImplementedAlert];
+    
+    XCTAssertEqualObjects(alert.title, @"Not Implemented");
+    XCTAssertEqualObjects(alert.message, @"This feature is not yet implemented.");
+    XCTAssertEqualObjects(alert.actions[0].title, @"Ok");
+}
+
+- (void)testAlertWithTitle
```

Comment:
Well, if you just want to test whether or not the block is called, I would use expectations rather than block variables, but I'm not sure if there's a particularly good way to programmatically trigger that action being called in a unit test.  But that's fine.  More what I meant was the exact same set of tests you have for when you pass a `nil` block, but just... pass a non-nil block.  The block doesn't matter.  And we're not trying to test here whether or not the block is called.  We're just asserting that whether or not we pass a closure, the title, message, and button title all get properly initialized.